### PR TITLE
Make Zookeeper initial timeout configurable

### PR DESCRIPTION
**Pull request by karlkfi**
30s is a little short sometimes.

### CODE REVIEW

File: records/config.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ type Config struct {
 	// Zookeeper: a single Zk url
 	Zk string
 
+	// Zookeeper Timeout: how long in seconds to wait for Zookeeper to be initially responsive (default 30)
+	ZkTimeout int
```
Comment (tsenart) [inline records/config.go]: Can we be more descriptive about which timeout this is configuring? `ZkConnectionTimeout`?

Comment (tsenart) [inline records/config.go]: Or, actually, `ZkDetectionTimeout`

File: main.go [hunk 2]
```diff
@@ -69,8 +65,10 @@ func main() {
 	}
 
 	reload := time.NewTicker(time.Second * time.Duration(config.RefreshSeconds))
-	timeout := time.AfterFunc(zkInitialDetectionTimeout, func() {
-		errch <- fmt.Errorf("master detection timed out after %s", zkInitialDetectionTimeout)
+
+	zkTimeout := time.Second * time.Duration(config.ZkTimeout)
```
Comment (tsenart) [inline main.go]: With my upcoming flag config change, all of these durations will be of type `time.Duration`. :-)

**Comment by tsenart** [comment]
Apart from the small naming comment, LGTM.

**Comment by karlkfi** [comment]
You want `ZkDetectionTimeout` or `bootstrapTimeout`?

**Comment by tsenart** [comment]
`ZkDetectionTimeout`

**Pull request merged**